homeassistant/components/ping/binary_sensor.py

```diff
@@ -104,6 +124,13 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
             ATTR_ROUND_TRIP_TIME_MIN: self._ping.data["min"],
         }
 
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if entity is enabled by default."""
+        if CONF_IMPORTED_BY in self.config_entry.options:
+            return bool(self.config_entry.options[CONF_IMPORTED_BY] == "binary_sensor")
+        return True
```

Comment:
You can set this as `self._attr_entity_registry_enabled_default`

I'm also not clear why you would create one if it would be disabled?